opencl/source/cl_device/cl_device_caps.cpp

```diff
@@ -198,6 +198,21 @@ void ClDevice::initializeCaps() {
         deviceExtensions += sharingFactory.getExtensions(driverInfo.get());
     }
 
+    PhysicalDevicePciBusInfo pciBusInfo;
+
+    if (driverInfo) {
```

Comment:
in what scenarios driverInfo  is null ? 
Why this if check is here?

Reply:
`driverInfo` can be null when the object of `ClDevice` class is used in unit tests.
The check is here because it is a place where the list of device extensions is filled.